pkg/sysprom/initialize.go

```diff
@@ -257,19 +257,18 @@ func createPrometheus(w *InitConfig) error {
 	serviceClient := w.client.CoreV1().Services(monitoringNS)
 	service := &apiv1.Service{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "prometheus",
+			Name:      "sys-prometheus",
```

Comment:
lets define sys-prometheus and sys-alertmanager as global constants.

Reply:
Done, reading all params from env variables instead of hard coding them.